apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -111,7 +31,7 @@ export class TeacherFeedback extends Component {
     teacher: PropTypes.number,
     verifiedTeacher: PropTypes.bool,
     displayKeyConcept: PropTypes.bool,
```

Comment:
Could you explain briefly how the component changes with displayKeyConcept = true vs. false?  (It seems to figure into more places than I expected below.)

Reply:
This is the comment in TopInstructions:
* The feedback tab will be the Key Concept tab if there is a mini rubric and:
* 1) Teacher is viewing the level but not giving feedback to the student
* 2) Student does not have any feedback for that level
* The Key Concept tab shows the Key Concept and Rubric for the level in a view
* only form
I'll take some time and see if I can do a refactor to make it more clear. Maybe add a `readOnly` prop.

Reply:
I took a really brief look just now and it's even more confusing than I had imagined.  :-)  I'm not sure if this helps or hurts, but what if we just created a KeyConcept component and displayed either TeacherFeedback or KeyConcept depending on the conditions above?

Reply:
That's a possibility. After poking around a bit I realized that the "Key Concept" is really a readonly rubric so I renamed `displayKeyConcept` to `displayReadonlyRubric`. I have a ticket in this Epic to rename "Key Concept" to rubric (https://codedotorg.atlassian.net/browse/LP-1904) so I'll revisit this again later.

Reply:
Let me know if the changes make it more clear what's going on, or if you'd rather I refactor into separate components

Reply:
Let's go with this for this PR and maybe we can have a quick chat about future direction?